src/containers/View3D/index.js

```diff
@@ -23,13 +23,13 @@ class View3D extends React.Component {
           frameborder="0"
           title="iFrame"
         />
-         <ButtonIcon
-              height="40px"
-              width="40px"
-              margin="20px"
-              icon={iconLeave}
-              action={this.props.shouldRender3DView}
-            />
+        <ButtonIcon
+          height="40px"
+          width="40px"
+          margin="20px"
+          icon={iconLeave}
+          action={this.props.shouldRender3DView}
```

Comment:
Je l'appelerais plutôt `show3DView`, de manière à ce que la méthode commence par un verbe à l'infinitif (`should` me semble plus approprié pour un booléen).